LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2365,6 +2365,91 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
+                            v.Add(new Voltage("3VSB", 7, 34, 34));
+                            v.Add(new Voltage("VBat", 8, 34, 34));
+                            v.Add(new Voltage("VTT", 9));
+                            //v.Add(new Voltage("#Unused #9", 9, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #10", 10, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #11", 11, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #12", 12, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #13", 13, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #14", 14, 0, 1, 0, true));
+
+                            t.Add(new Temperature("CPU", 1));
+                            t.Add(new Temperature("MotherBoard", 2));
+                            //t.Add(new Temperature("AUX 0", 3)); //No software from Asus reports this temperature ~82C
+                            //t.Add(new Temperature("#Unused 4", 4));
+                            //t.Add(new Temperature("#Unused 5", 5));
+                            //t.Add(new Temperature("#Unused 6", 6));
+                            //t.Add(new Temperature("#Unused 7", 7));
+                            //t.Add(new Temperature("#Unused 8", 8));
+                            //t.Add(new Temperature("#Unused 9", 9));
+                            //t.Add(new Temperature("#Unused 10", 10));
+                            t.Add(new Temperature("PCH Chip CPU Max", 11));
+                            t.Add(new Temperature("PCH Chip", 12));
+                            t.Add(new Temperature("PCH CPU", 13));
+                            t.Add(new Temperature("PCH MCH", 14));
+                            t.Add(new Temperature("Agent 0 DIMM 0", 15));
+                            //t.Add(new Temperature("Agent 0 DIMM 1", 16));
+                            t.Add(new Temperature("Agent 1 DIMM 0", 17));
+                            //t.Add(new Temperature("Agent 1 DIMM 1", 18));
+                            t.Add(new Temperature("Device 0", 19));
+                            t.Add(new Temperature("Device 1", 20));
+                            t.Add(new Temperature("PECI 0 Calibrated", 21));
+                            t.Add(new Temperature("PECI 1 Calibrated", 22));
+                            t.Add(new Temperature("Virtual", 23));
+
+                            //todo
+                            //Find a way to read the value of VRM HS FAN
+                            //It is definitely not controllable as it only supports ON / OFF options but the RPM can be read somewhere
+
+                            for (int i = 0; i < superIO.Fans.Length; i++)
+                            {
+                                switch (i)
+                                {
+                                    case 0:
+                                        f.Add(new Fan("Chassis fan", 0));
```

Comment:
Fan (also below)

Reply:
I just wanted to keep the names according to AI Suite for easier understanding for users

Reply:
It should be Chassis **F**an (with a capital) to follow naming in app :) 
Same applies to Motherboard.

Reply:
![image](https://user-images.githubusercontent.com/13592821/128633731-b090bbb6-bfcb-4b27-821c-2394e5077c33.png)

Reply:
All good now to merge?

Reply:
I think so, it looks fine to me
![image](https://user-images.githubusercontent.com/13592821/128689891-815765e0-5e45-416d-ab36-498f983abedd.png)